terasoluna-tourreservation-selenium/pom.xml

```diff
@@ -2,20 +2,27 @@
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
     xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
     <modelVersion>4.0.0</modelVersion>
-    <parent>
-        <version>5.0.3-SNAPSHOT</version>
-        <artifactId>terasoluna-tourreservation-parent</artifactId>
-        <groupId>org.terasoluna.app</groupId>
-        <relativePath>../terasoluna-tourreservation-parent/pom.xml</relativePath>
-    </parent>
+
     <artifactId>terasoluna-tourreservation-selenium</artifactId>
+    <packaging>jar</packaging>
     <name>TERASOLUNA Server Framework for Java (5.x) - Tour Reservation Application - Selenium</name>
     <description>Selenium tests for Tour Reservation Application using TERASOLUNA Server Framework for Java (5.x)</description>
-    <packaging>jar</packaging>
+
+    <parent>
+        <groupId>org.terasoluna.app</groupId>
+        <artifactId>terasoluna-tourreservation</artifactId>
+        <version>5.0.3-SNAPSHOT</version>
+        <relativePath>../pom.xml</relativePath>
+    </parent>
 
     <dependencies>
         <dependency>
             <groupId>${project.groupId}</groupId>
+            <artifactId>terasoluna-tourreservation-env</artifactId>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
+            <groupId>${project.groupId}</groupId>
```

Comment:
- [ ] Why you changed dependency? I think that it is sufficient to change the project structure.
If you change this, please see and consider to change 5.1 and later too.